politeiawww/cmd/politeiawwwcli/config/config.go

```diff
@@ -36,7 +36,7 @@ var (
 	defaultWalletCertFile = filepath.Join(dcrwalletHomeDir, "rpc.cert")
 
 	Host       = defaultHost
-	HomeDir    = cleanAndExpandPath(defaultHomeDir)
+	HomeDir    = util.CleanAndExpandPath(defaultHomeDir)
```

Comment:
This should just be `HomeDir = defaultHomeDir`.  This was a mistake on my part when I wrote it.  I was planning on replacing the defaultHomeDir variable with the input from a dataDir flag, but never did.